en_US.ini

```diff
@@ -598,10 +598,10 @@ Force real clock sync (slower, less lag) = Force real clock sync (slower, less l
 General = General
 Help the PPSSPP team = Help the PPSSPP team
 Host (bugs, less lag) = Host (bugs, less lag)
-I/O on thread (experimental) = I/O on thread (experimental)
+I/O on thread (experimental) = I/O on thread
 IO timing method = I/O timing method
 MMDDYYYY = MMDDYYYY
-Multithreaded (experimental) = Multithreaded (experimental)
+Multithreaded (experimental) = Multithreaded (glitches)
```

Comment:
Well, it causes glitches in some cases.  I'm trying to avoid verbosity here, but "may cause glitches" may fit fine...

We've talked about some ways to multithread differently, and Vulkan may also bring better opportunities.

Ultimately, any of these options are just going to be new features.  As it is, the current option has the problem that if your cpu gets too far ahead of your gpu (this will often happen), things will go wrong.  Solutions to that basically center on (a) synchronize so often that the option is not faster or is actually slower or (b) do more work to pre-calculate synchronization, which it seems like will mostly negate the speed gains.  (b) looks like the best option right now but also a big pain.

I had even tried to jit the gpu to possibly find some gains and maybe be able to cache sync times or who knows, but that wasn't faster either.

Probably if we get gpu/cpu multithreading working better, it will mean new experimental code, and a new actually "experimental" option.

But can leave it "experimental" for now and change it later...

-[Unknown]